docs/pages/messages/turbo-json-parse-error.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: `turbo.json` Parse Error
```

Comment:
We use sentence casing so "`turbo.json` parse error".

(Well, at least we're supposed to. This is one of many things that will get cleaned in our docs on the `docs-new` branch.

Reply:
I'm gonna drop this one too tbh. The error will be more descriptive than this page because it'll actually show the real syntax error

Reply:
Beauuuuuty. I was wondering if that was going to be the case.